### ISSUE TITLE
Steam cliemt gets stuck on "Logging in" in Nobara 38.

### ISSUE THREAD

**Issue by JustSkotch** [ValveSoftware/steam-for-linux#10534]
#### system information
* Steam client version (build number or date): 1705108172
* Distribution (e.g. Ubuntu): Nobara 38 (fork of Fedora)
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes, everything is up to date.
* Steam Logs:  [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/14403580/steam-logs.tar.gz)
* GPU: Nvidia

#### Please describe your issue in as much detail as possible:
It just won't log me in, bootstrap, cef and connection logs show some errors.

#### Steps for reproducing this issue:
1. Launch the Steam (With file or through console).
2. The window pops up and shows "Logging in" with a throbber and my profile picture in center.
3. Nothing changes no matter how long I wait.

**Comment by kisak-valve** [comment]
Hello @JustSkotch, if you completely close Steam, then run `steam` from a terminal, does that have an effect or show any additional hints in the terminal spew?

**Comment by JustSkotch** [comment]
> Hello @JustSkotch, if you completely close Steam, then run `steam` from a terminal, does that have an effect or show any additional hints in the terminal spew?

Not really. However, what I've found out is that running two commands seems to solve the issue but only for one start.
`sudo systemctl restart NetworkManager`
and
`sudo update-ca-trust`
I also had reoccurring problem with an SSL certificate for steamcommunity but only in the client, maybe those two are somewhat related.
